Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "libc"
-version = "0.2.139"
+version = "0.2.140"
```

Comment:
Could you drop this? As mentioned in https://github.com/rust-lang/libc/blob/master/.github/PULL_REQUEST_TEMPLATE.md, A version bump PR shouldn't contain any other changes to avoid blocking a release when a review takes time.

Reply:
Sure, I misunderstood that statement and thought that a PR should contian a version bump as a commit without any other changes.